integrations/Podsights/constants.js

```diff
@@ -0,0 +1,160 @@
+const NAME = "PODSIGHTS";
+const CNameMapping = {
+  [NAME]: NAME,
+  Podsights: NAME,
+  PodSights: NAME,
+  "pod Sights": NAME,
+  "Pod Sights": NAME,
+  "pod sights": NAME,
+  "POD SIGHTS": NAME,
+  "Pod sights": NAME,
+};
+
+const LEAD_EVENT = [
+  {
+    destKey: "value",
+    sourceKeys: "value",
+  },
+  {
+    destKey: "currency",
+    sourceKeys: "currency",
+  },
+  {
+    destKey: "type",
+    sourceKeys: "type",
+  },
+  {
+    destKey: "category",
+    sourceKeys: "category",
+  },
+];
+
+const PRODUCT_EVENT = [
+  {
+    destKey: "value",
+    sourceKeys: "value",
+  },
+  {
+    destKey: "currency",
+    sourceKeys: "currency",
+  },
+  {
+    destKey: "product_id",
+    sourceKeys: "productId",
+  },
+  {
+    destKey: "product_name",
+    sourceKeys: "productName",
+  },
+  {
+    destKey: "product_type",
+    sourceKeys: "productType",
+  },
+  {
+    destKey: "product_vendor",
+    sourceKeys: "productVendor",
+  },
+];
+
+const ADD_TO_CART_EVENT = [
+  {
+    destKey: "value",
+    sourceKeys: "value",
+  },
+  {
+    destKey: "currency",
+    sourceKeys: "currency",
+  },
+  {
+    destKey: "product_id",
+    sourceKeys: "productId",
+  },
+  {
+    destKey: "product_name",
+    sourceKeys: "productName",
+  },
+  {
+    destKey: "product_type",
+    sourceKeys: "productType",
+  },
+  {
+    destKey: "product_vendor",
+    sourceKeys: "productVendor",
+  },
+  {
+    destKey: "quantity",
+    sourceKeys: "quantity",
+  },
+  {
+    destKey: "variant_id",
+    sourceKeys: "variantId",
+  },
+  {
+    destKey: "variant_name",
+    sourceKeys: "variantName",
+  },
+];
+
+const CHECK_OUT_EVENT = [
+  {
+    destKey: "value",
+    sourceKeys: "value",
+  },
+  {
+    destKey: "currency",
+    sourceKeys: "currency",
+  },
+  {
+    destKey: "discount_code",
+    sourceKeys: "discountCode",
+  },
+  {
+    destKey: "quantity",
+    sourceKeys: "quantity",
+  },
+  {
+    destKey: "line_items",
+    sourceKeys: "lineItems",
+  },
+];
+
+const PURCHASE_EVENT = [
+  {
+    destKey: "value",
+    sourceKeys: "value",
+  },
+  {
+    destKey: "currency",
+    sourceKeys: "currency",
+  },
+  {
+    destKey: "discount_code",
+    sourceKeys: "discountCode",
+  },
+  {
+    destKey: "line_items",
+    sourceKeys: "lineItems",
```

Comment:
Line items should contain product object if any. This is what we see from their documentation:
```
pdst('checkout', {
  value: 10.0, // price
  currency: 'USD',
  discount_code: 'PODCAST_CODE',

  // optional line_items, if you would like advanced analytics.
  quantity: 3,
  line_items: [
    {
      value: 21.2,
      quantity: 1,
      product_id: '{{product.id}}',
      product_name: '{{product.title}}',
      product_type: '{{product.type}}',
      product_vendor: '{{product.vendor}}',
      variant_id: '{{variant.id}}',
      variant_name: '{{variant.title}}',
    },
    {
      value: 42.4,
      quantity: 2,
      product_id: '{{product.id}}',
      product_name: '{{product.title}}',
      product_type: '{{product.type}}',
      product_vendor: '{{product.vendor}}',
      variant_id: '{{variant.id}}',
      variant_name: '{{variant.title}}',
    },
  ],});
```
check what happens with these events if line_items are not sent

Reply:
`line_items` is not visible in the debugger section for any event, and it doesn't cause any effect if we pass it or not.

Reply:
As mentioned in the docs, should we take `line_items` object values from `message.properties`?

Reply:
if product array is present, then from message.properties.products if not then message.properties

Reply:
Okay, got it, will update